python/metatensor-torch/metatensor/torch/atomistic/model.py

```diff
@@ -483,6 +483,19 @@ def save(self, file: Union[str, Path], collect_extensions: Optional[str] = None)
         except RuntimeError as e:
             raise RuntimeError("could not convert the module to TorchScript") from e
 
+        if self._capabilities.length_unit == "":
+            warnings.warn(
+                "No length unit was provided for the model.",
+                stacklevel=1,
+            )
+
+        for name, output in self._capabilities.outputs.items():
+            if output.unit == "":
+                warnings.warn(
+                    f"No units were provided for output {name}.",
+                    stacklevel=1,
+                )
```

Comment:
There are some outputs where no units is expected ("features" output), and then we should not warn. 

At the same time, non-standard outputs (anything with `::` inside the name) might also not require units

Reply:
fair, do we want to check only for a certain set of outputs then? (energy, forces, dipoles, ...)

Reply:
yes, I would only check for "standard outputs" (like we already do with `check_consistency=True`)

Reply:
addressed with a custom list of standard outputs for now, and left a TODO in case we want to standardize this across the code system